recipes/eleuther_eval.py

```diff
@@ -153,10 +158,16 @@ def _setup_model(
         with utils.set_default_dtype(self._dtype), self._device:
             model = config.instantiate(model_cfg)
 
-        model.load_state_dict(model_state_dict)
+        if self._quantization_mode is not None:
+            quantizer = utils.get_quantizer(self._quantization_mode)
+            model = quantizer.quantize(model)
+
+        model.load_state_dict(model_state_dict, assign=True)
```

Comment:
I'm trying to understand the implication of ```assign=True```, mind elaborating on why we need this?

Reply:
without assign it is using param.copy_ which doesn't seem to be supported for tensor subclass I think